<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileDefDialog</class>
 <widget class="QMainWindow" name="TileDefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tile Properties</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="tilesNtilesetLayout">
        <item>
         <layout class="QVBoxLayout" name="tilesetLayout">
          <item>
           <widget class="QListWidget" name="tilesets">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="toolBarLayout"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="tilesLayout">
          <item>
           <widget class="Tiled::Internal::MixedTilesetView" name="tiles"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="enums"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="scaleComboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="propertiesLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-156</y>
             <width>263</width>
             <height>1282</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="propertySheet" native="true">
              <layout class="QFormLayout" name="formLayout_4">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string>Collision Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="CollideNorth">
                    <property name="text">
                     <string>CollideNorth</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="CollideWest">
                    <property name="text">
                     <string>CollideWest</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Door Functions</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Door</string>
                    </property>
                    <property name="buddy">
                     <cstring>Door</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="Door"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>DoorFrame</string>
                    </property>
                    <property name="buddy">
                     <cstring>DoorFrame</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="DoorFrame"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Extra Functions</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_3">
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="IsBed">
                    <property name="text">
                     <string>IsBed</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QGroupBox" name="groupBox_5">
                 <property name="title">
                  <string>Floor Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_6">
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="FloorOverlay">
                    <property name="text">
                     <string>FloorOverlay</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="IsFloor">
                    <property name="text">
                     <string>IsFloor</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QCheckBox" name="IsIndoor">
                    <property name="text">
                     <string>IsIndoor</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QGroupBox" name="groupBox_6">
                 <property name="title">
                  <string>Roof Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_7">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>RoofStyle</string>
                    </property>
                    <property name="buddy">
                     <cstring>RoofStyle</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="RoofStyle"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="title">
                  <string>Sheet Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_8">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                  </property>
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="ClimbSheetN">
                    <property name="text">
                     <string>ClimbSheetN</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="ClimbSheetW">
                    <property name="text">
                     <string>ClimbSheetW</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="12" column="0">
                <widget class="QGroupBox" name="groupBox_4">
                 <property name="title">
                  <string>Wall Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_5">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                  </property>
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="HoppableN">
                    <property name="text">
                     <string>HoppableN</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="HoppableW">
                    <property name="text">
                     <string>HoppableW</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QCheckBox" name="WallOverlay">
                    <property name="text">
                     <string>WallOverlay</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>WallStyle</string>
                      </property>
                      <property name="buddy">
                       <cstring>WallStyle</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="WallStyle"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="9" column="0">
                <widget class="QGroupBox" name="groupBox_8">
                 <property name="title">
                  <string>Shelf Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_9">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>FloorItemShelf</string>
                    </property>
                    <property name="buddy">
                     <cstring>FloorItemShelf</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="FloorItemShelf"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>HighItemShelf</string>
                    </property>
                    <property name="buddy">
                     <cstring>HighItemShelf</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="HighItemShelf"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>TableItemShelf</string>
                    </property>
                    <property name="buddy">
                     <cstring>TableItemShelf</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QComboBox" name="TableItemShelf"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="10" column="0">
                <widget class="QGroupBox" name="groupBox_9">
                 <property name="title">
                  <string>Stair Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_10">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>StairStyle</string>
                    </property>
                    <property name="buddy">
                     <cstring>StairStyle</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="StairStyle"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="11" column="0">
                <widget class="QGroupBox" name="groupBox_10">
                 <property name="title">
                  <string>Visibility Functions</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_11">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                  </property>
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="PreSeen">
                    <property name="text">
                     <string>PreSeen</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="13" column="0">
                <widget class="QGroupBox" name="groupBox_11">
                 <property name="title">
                  <string>Water Functions</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_12">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                  </property>
                  <item row="2" column="0">
                   <widget class="QCheckBox" name="WaterPiped">
                    <property name="text">
                     <string>Water Piped</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>WaterAmount</string>
                      </property>
                      <property name="buddy">
                       <cstring>WaterAmount</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="WaterAmount"/>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>WaterMaxAmount</string>
                      </property>
                      <property name="buddy">
                       <cstring>WaterMaxAmount</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="WaterMaxAmount"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="14" column="0">
                <widget class="QGroupBox" name="groupBox_12">
                 <property name="title">
                  <string>Window Functions</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_13">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                  </property>
                  <item row="3" column="0">
                   <widget class="QCheckBox" name="WindowLocked">
                    <property name="text">
                     <string>WindowLocked</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string>OpenTileOffset</string>
                      </property>
                      <property name="buddy">
                       <cstring>OpenTileOffset</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="OpenTileOffset"/>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>SmashedTileOffset</string>
                      </property>
                      <property name="buddy">
                       <cstring>SmashedTileOffset</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="SmashedTileOffset"/>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QLabel" name="label_11">
                      <property name="text">
                       <string>Window</string>
                      </property>
                      <property name="buddy">
                       <cstring>Window</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="Window"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QGroupBox" name="groupBox_13">
                 <property name="title">
                  <string>Object Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_14">
                  <property name="fieldGrowthPolicy">
                   <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                  </property>
                  <item row="1" column="0">
                   <widget class="QCheckBox" name="WheelieBin">
                    <property name="text">
                     <string>WheelieBin</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_12">
                    <item>
                     <widget class="QLabel" name="label_15">
                      <property name="text">
                       <string>ContainerType</string>
                      </property>
                      <property name="buddy">
                       <cstring>ContainerType</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="ContainerType">
                      <property name="minimumSize">
                       <size>
                        <width>128</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="editable">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QGroupBox" name="groupBox_14">
                 <property name="title">
                  <string>Global Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_15">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>TileBlockStyle</string>
                    </property>
                    <property name="buddy">
                     <cstring>TileBlockStyle</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="TileBlockStyle"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QGroupBox" name="groupBox_15">
                 <property name="title">
                  <string>Light Settings</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_16">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>LightPolyStyle</string>
                    </property>
                    <property name="buddy">
                     <cstring>LightPolyStyle</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox" name="LightPolyStyle"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionAddTileset"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionAddTileset">
   <property name="text">
    <string>Add Tilesets...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::MixedTilesetView</class>
   <extends>QTableView</extends>
   <header>BuildingEditor/mixedtilesetview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
